DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/pipelined_kernels/sample.json

```diff
@@ -1,71 +0,0 @@
-{
```

Comment:
If removing the pipelined_kernels sample, it needs to be removed from the top level readme https://github.com/oneapi-src/oneAPI-samples/blob/development/DirectProgramming/C%2B%2BSYCL_FPGA/README.md

Reply:
removed. i noticed that invocation_interfaces was missing as well, so i added it